src/components/ListDetails/ListDetails.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import * as S from './Style'
+import { serviceNode } from '../../services/node/node'
+
+import ItemDetails from '../ItemDetails/ItemDetails'
+import Loader from '../Loader/Loader'
+import Status from '../Status/Status'
+
+const ListDetails = ({ }) => {
+  const [expenseList, setExpenseList] = useState(undefined)
+  
+
+  useEffect(() => {
```

Comment:
Uzyj async await

Reply:
na callbacku useEffect nie zadziała, ale możesz to zrobić wewnątrz IIFE patrz Czarka PR

Reply:
done